src/app/shared/wysiwyg/wysiwyg.component.ts

```diff
@@ -106,7 +108,10 @@ export class WysiwygComponent implements OnInit, OnChanges, OnDestroy {
     return this.url && this.url.invalid && this.url.dirty;
   }
 
-  promptForLink() {
+  promptForLink(url?: string, title?: string) {
+    this.linkURL = url;
+    this.linkTitle = title;
+    this.chgRef.detectChanges();
```

Comment:
seems to have introduced an Expression Changed After Checked Error if the user hits the Enter button instead of clicking Okay on the prompt